src/DynamoCoreWpf/ViewModels/Preview/ConnectorAnchorViewModel.cs

```diff
@@ -244,10 +246,13 @@ internal void DisposeViewModel()
         /// <param name="connectorViewModel"></param>
         /// <param name="dynamoModel"></param>
         /// <param name="tooltipText"></param>
-        public ConnectorAnchorViewModel(ConnectorViewModel connectorViewModel, DynamoModel dynamoModel, string tooltipText)
+        public ConnectorAnchorViewModel(ConnectorViewModel connectorViewModel,
+           DynamoViewModel dynamoViewModel,
+           string tooltipText)
```

Comment:
Just so whoever watching is aware, this is NOT API breaking changes because this new API only exist in Dynamo 2.13